input/fsh/DkCoreOrganization.fsh

```diff
@@ -115,7 +118,18 @@ Usage: #example
 * identifier[=].value = "urn:uuid:a107fd1e-9f35-422c-9d1e-add097bbf4a6"
 * type = $sct#264372000
 
+Instance: 154b8c96-a061-45bf-9ce4-1947c7c3c283
+InstanceOf: DkCoreOrganization
+Title: "Producer Test Organization"
+Description: "Producer of Lab Results Test Organization"
+Usage: #example
+* identifier[0].system = "urn:oid:1.2.208.176.1.1"
+* identifier[=].value = "12345678901"
```

Comment:
This should be a valid SOR Identifier

Reply:
Can be validated using https://sundhedsdatastyrelsen.dk/da/rammer-og-retningslinjer/organisationsregistrering